lib/api_me.rb

```diff
@@ -165,8 +165,11 @@ def filter_scope(scope)
     ).results
   end
 
-  def sort_scope(scope, params)
-    ApiMe::Sorting.new(scope: scope, sort_params: params).results
+  def sort_scope(scope)
+    sort_klass.new(
```

Comment:
This seems to be a problem
```
Failure/Error: get '/api/v1/posts'
     NameError:
       undefined local variable or method `sort_klass' for #<Api::V1::PostsController:0x00000002f0bb88>
```
@Samsinite is this because the var could be null?